web/README.md

```diff
@@ -0,0 +1,29 @@
+# web
```

Comment:
Maybe once we get all of the other details worked out we can add some more documentation in the main docs repo for this.